src/standard/fields.js

```diff
@@ -50,7 +50,7 @@ const fromTemplate = (template, options) => box => {
 
   const makeText = (x, y, text) => ({
     type: 'text', x, y, text,
-    'dominant-baseline': 'hanging'
```

Comment:
`dominant-baseline` outlived its usefulness for quite some time now.
Due to different layout algorithms, icon bounding boxes always never match their milsymbol counterparts.
For this reason vertical alignment of text boxes is notoriously off by a few pixels. In most cases it actually "looks" better than the original.

I will remove `dominant-baseline` in the final commit.